server/composer.json

```diff
@@ -1,5 +1,8 @@
 {
     "require": {
         "slim/slim": "^3.0"
-    }
+    },
+		"scripts": {
```

Comment:
Check indentation. Regardless, what's being done in ./bin/build should not be part of post-install. Symlinks should be a concern of the `web` container build and not of composer.

Reply:
@dayllanmaza I don't understand the reasoning, it's a post install step. this is the same thing that is in `client/package.json`. We __could__ make the web container responsible for building (by running `composer install` before starting lighttpd).

Reply:
Yes we could have the web container do composer install, I don't think it matters in this case since it is gonna be the same image anyway. Regarding the script, one of the reason is that composer install runs in prod and this is only valid in dev

Reply:
well I mean I think it should be consistent with `client`, but the web container does not have `npm`. :/ don't you manually have to run `composer install` in prod anyways?

Reply:
nvm. I'm going to change it it makes more sense since the container stays running.

Reply:
What if we moved this into `./bin/start`? if you think about it this is more of a requirement for that container to be successfully launched than it is for composer to successfully install dependencies. Basically, this is a requirement of the container to be able to function and not of the `app`.